app/controllers/workflow_repository_controller.rb

```diff
@@ -16,10 +16,75 @@ def self.display_methods
     %w[workflows]
   end
 
+  def self.custom_display_modes
+    %w[output]
+  end
+
   def self.model
     ManageIQ::Providers::Workflows::AutomationManager::ConfigurationScriptSource
   end
 
+  def show_searchbar?
```

Comment:
I think a lot of these methods should be made private if they are not routable - can you ensure they are made private where appropriate?

Reply:
I tried to make them private and it just broke functionality. Not sure if any of these can be made private.